src/deployments/inputs/HorizonPhaseOneListing.sol

```diff
@@ -0,0 +1,646 @@
+  uint256 public immutable GHO_INITIAL_DEPOSIT;
+
+  address public immutable USDC_ADDRESS;
+  address public immutable USDC_PRICE_FEED;
+  uint256 public immutable USDC_INITIAL_DEPOSIT;
+
+  address public immutable RLUSD_ADDRESS;
+  address public immutable RLUSD_PRICE_FEED;
+  uint256 public immutable RLUSD_INITIAL_DEPOSIT;
+
+  address public immutable USTB_ADDRESS;
+  address public immutable USTB_PRICE_FEED_ADAPTER;
+
+  address public immutable USCC_ADDRESS;
+  address public immutable USCC_PRICE_FEED_ADAPTER;
+
+  address public immutable USYC_ADDRESS;
+  address public immutable USYC_PRICE_FEED;
+
+  address public immutable JTRSY_ADDRESS;
+  address public immutable JTRSY_PRICE_FEED_ADAPTER;
+
+  address public immutable JAAA_ADDRESS;
+  address public immutable JAAA_PRICE_FEED_ADAPTER;
+
+  bytes32 public constant EMERGENCY_ADMIN_ROLE = keccak256('EMERGENCY_ADMIN');
+
+  constructor(MarketReport memory report) AaveV3Payload(IEngine(report.configEngine)) {
+    ATOKEN_IMPLEMENTATION = report.aToken;
+    RWA_ATOKEN_IMPLEMENTATION = report.rwaAToken;
+    VARIABLE_DEBT_TOKEN_IMPLEMENTATION = report.variableDebtToken;
+
+    ACL_MANAGER = ACLManager(report.aclManager);
+    require(report.poolConfiguratorProxy == address(CONFIG_ENGINE.POOL_CONFIGURATOR()));
+    CONFIGURATOR = IPoolConfigurator(report.poolConfiguratorProxy);
+    POOL = IPool(report.poolProxy);
+
+    // https://github.com/bgd-labs/aave-address-book/blob/1a3c41b8b267fe9ef994df6427f767c8d4f08900/src/AaveV3Ethereum.sol#L108
+    DUST_BIN = 0x31a0Ba3C2242a095dBF58A7C53751eCBd27dBA9b;
+
+    GHO_ADDRESS = 0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f;
+    GHO_PRICE_FEED = 0xD110cac5d8682A3b045D5524a9903E031d70FCCd;
+    GHO_INITIAL_DEPOSIT = 100 * (10 ** IERC20Detailed(GHO_ADDRESS).decimals());
+
+    USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    USDC_PRICE_FEED = 0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6;
+    USDC_INITIAL_DEPOSIT = 100 * (10 ** IERC20Detailed(USDC_ADDRESS).decimals());
+
+    RLUSD_ADDRESS = 0x8292Bb45bf1Ee4d140127049757C2E0fF06317eD;
+    RLUSD_PRICE_FEED = 0x26C46B7aD0012cA71F2298ada567dC9Af14E7f2A;
+    RLUSD_INITIAL_DEPOSIT = 100 * (10 ** IERC20Detailed(RLUSD_ADDRESS).decimals());
+
+    USTB_ADDRESS = 0x43415eB6ff9DB7E26A15b704e7A3eDCe97d31C4e;
+    USTB_PRICE_FEED_ADAPTER = 0x5Ae4D93B9b9626Dc3289e1Afb14b821FD3C95F44;
+
+    USCC_ADDRESS = 0x14d60E7FDC0D71d8611742720E4C50E7a974020c;
+    USCC_PRICE_FEED_ADAPTER = 0x14CB2E810Eb93b79363f489D45a972b609E47230;
+
+    USYC_ADDRESS = 0x136471a34f6ef19fE571EFFC1CA711fdb8E49f2b;
+    USYC_PRICE_FEED = 0xE8E65Fb9116875012F5990Ecaab290B3531DbeB9;
+
+    JTRSY_ADDRESS = 0x8c213ee79581Ff4984583C6a801e5263418C4b86;
+    JTRSY_PRICE_FEED_ADAPTER = 0xfAB6790E399f0481e1303167c655b3c39ee6e7A0;
+
+    JAAA_ADDRESS = 0x5a0F93D040De44e78F251b03c43be9CF317Dcf64;
+    JAAA_PRICE_FEED_ADAPTER = 0xF77f2537dba4ffD60f77fACdfB2c1706364fA03d;
+  }
+
+  function eModeCategoriesUpdates()
```

Comment:
i think we can use `eModeCategoriesCreations()` instead, wdyt? 
ref https://github.com/bgd-labs/aave-proposals-v3/pull/755/files#diff-cac05e5cd437fbcd8e4873b202c927d859e1943a4cffce9d815718845b5f228eR61-R89
nbd either way

Reply:
this was added after the snapshot of this repo

https://github.com/aave-dao/aave-v3-origin/commit/19a1531333905def5d80edb3b1009877a287dd02

hence we don't have this change in this repo